### ISSUE TITLE
Support writing different models to one pdb file

### ISSUE THREAD

**Issue by danijoo** [biotite-dev/biotite#131]
The pdb specification allows pdb files with multiple different models. biotite can read these files, but there is no way to write them.

**Comment by padix-key** [comment]
You are right, this is definitely something *Biotite* should support. However, solving this issue requires quite a bunch of work, so it may take some time until I get to fix this.

**Comment by ncoish** [comment]
Is this issue still a problem? When I try to write an AtomArrayStack to a pdb file I correctly get all models written to the file.

**Comment by padix-key** [comment]
This issue is about writing different models in terms of different atoms per model (e.g. an additional atom), not just different coordinates. In other words the problem is that a `list` of `AtomArray` cannot be given to `set_structure()`, although the format specification would allow it.

**Comment by padix-key** [comment]
The issue title is a bit misleading, so I will clarify it.

**Comment by ncoish** [comment]
> This issue is about writing different models in terms of different atoms per model (e.g. an additional atom), not just different coordinates. In other words the problem is that a `list` of `AtomArray` cannot be given to `set_structure()`, although the format specification would allow it.

Understood. Thanks for the clarification!
